src/ParseObject.js

```diff
@@ -131,9 +132,10 @@ export default class ParseObject {
   /** Prototype getters / setters **/
 
   get attributes(): AttributeMap {
-    return Object.freeze(
-      ObjectState.estimateAttributes(this.className, this._getStateIdentifier())
-    );
+    let attributes = singleInstance ?
```

Comment:
@hallucinogen I thought about that too, but they take completely different arguments -- one uses a set of strings as identifiers, the other uses an object. We could pass the Object and introspect the identifiers in most, but not all of the cases